src/nvim/decoration.h

```diff
@@ -34,6 +34,10 @@ typedef struct {
   int start_col;
   int end_row;
   int end_col;
+  int ordering;
+  DecorPriority priority;
+  bool owned;  ///< ephemeral decoration, free memory immediately
+  DecorRangeKind kind: 8;
```

Comment:
We don't use bitfields in our owned code, better to change the typedef to `typedef uint8_t DecorRangeKind`